### PR TITLE
Rep stamps (attempt 2)

### PR DESCRIPTION
**Pull request by Loldaddy5676**
## About The Pull Request
Replaces captains, centcom, chaplin, and cmo stamps for fourth pack, K-corp, Hana, and W-corp stamps.

## Why It's Good For The Game
Paperwork RP

## Changelog
:cl:
Replaces four ss13 stamps with Fourth pack, K-corp, W-corp, and hana stamps
/:cl:

### CODE REVIEW

**Review by Kitsunemitsu — COMMENTED** [review]
Shouldn't you make your own stamps file?

**Comment by Loldaddy5676** [comment]
> Shouldn't you edit the old stamp code?

pretty sure i did, i only changed the names for four of the stamps.

File: stamps.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/obj/item/stamp
```
Comment (Kitsunemitsu) [inline stamps.dm:1]: Says here you made a new stamp.dm file.

Comment (Loldaddy5676) [reply stamps.dm:1]: gonna fix it once i got time

**Comment by Kitsunemitsu** [comment]
Reopen if you want to work on it again.

**Pull request closed without merging**